test/Streamly/Test/Data/Array.hs

```diff
@@ -8,4 +8,43 @@
 
 module Streamly.Test.Data.Array where
 
-#include "Streamly/Test/Common/Array.hs"
+#include "Streamly/Test/Data/Array/CommonImports.hs"
+
+import qualified Streamly.Internal.Data.Array as A
+type Array = A.Array
```

Comment:
We can just import Array instead of defining this type.

Reply:
Can remove it. A lot of things need to be cleaned up. I'll defer this to cleanup.